charts/kube-prometheus-stack/values.yaml

```diff
@@ -2215,6 +2215,10 @@ prometheusOperator:
     tag: v0.30.2
     sha: ""
 
+  ## Set a Label Selector to filter watched alertManager and thanosRuler
+  alertManagerSelector: ""
+  thanosRulerSelector: ""
```

Comment:
Can you also add this one prometheus-instance-selector? https://github.com/prometheus-operator/prometheus-operator/blob/27943fe43862d313a63b518b6ee67b59d7680c1d/cmd/operator/main.go#L175

Reply:
Done